### PR TITLE
+mparser 1.1

### PR DESCRIPTION
**Pull request by cakeplus**

### CODE REVIEW

**Comment by cakeplus** [comment]
One of the build jobs failed because of the temporarily unavailable Ubuntu mirror. It is safe to merge the PR.

**Comment by yallop** [comment]
I've restarted the failed job.

File: packages/mparser/mparser.1.1/opam [hunk 1]
```diff
@@ -0,0 +1,18 @@
+opam-version: "1.1"
+maintainer: "Max Mouratov <mmouratov@gmail.com>"
+homepage: "https://github.org/cakeplus/mparser/"
```
Comment (dbuenzli) [inline packages/mparser/mparser.1.1/opam]: This URL is wrong `.org` -> `.com`.

Comment (cakeplus) [inline packages/mparser/mparser.1.1/opam]: Well spotted, thank you! Fixed it.

File: packages/mparser/mparser.1.1/opam [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+license: "LGPL-2.1 with OCaml linking exception"
+build: [
+  ["ocaml" "setup.ml" "-configure" "--prefix" prefix "--%{re:enable}%-re" "--%{pcre:enable}%-pcre"]
+  ["ocaml" "setup.ml" "-build"]
+  ["ocaml" "setup.ml" "-install"]
+]
+remove: [["ocaml" "setup.ml" "uninstall"]]
+depends: [
+  "ocamlfind"
+  "pcre"
```
Comment (avsm) [inline packages/mparser/mparser.1.1/opam]: pcre is in both the `depends` and `depopts`

Comment (cakeplus) [inline packages/mparser/mparser.1.1/opam]: Indeed. Thank you.
Guess I need to fix my sleeping patterns :-(

**Pull request merged**